package.json

```
@@ -30,7 +30,9 @@
   },
   "scripts": {
     "build": "node lib/transform.js",
-    "test": "c8 mocha"
+    "build-java": "javac java/Validator.java",
+    "test": "c8 mocha --reporter-option maxDiffSize=0",
+    "test-xsd": "java -cp java Validator"
```

We could combine the two test steps:
```suggestion
    "test": "npm run test-js && npm run test-xsd",
    "test-js": "c8 mocha --reporter-option maxDiffSize=0",
    "test-xsd": "java -cp java Validator"
```
Same for the two build steps, then `nodejs.yml` doesn't have to be changed.